### PR TITLE
Update basic-features.md for v5.3

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: basic-features.md [hunk 1]
```diff
@@ -10,166 +10,166 @@ aliases: ['/docs-cn/dev/basic-features/']
...
-| [通过系统变量 tidb_snapshot 读取历史数据](/read-historical-data.md)                                         | Y            | Y            | Y            | Y            |
-| [Optimizer hints](/optimizer-hints.md)                                                                   | Y            | Y            | Y            | Y            |
-| [MPP 执行引擎](/explain-mpp.md)                                                                   | Y            | Y            | Y            | N            |
-| [Index Merge Join](/explain-index-merge.md)                                                              | 实验特性 | 实验特性 | 实验特性 | 实验特性 |
+| 高级 SQL 功能                                                                                | 5.3          | 5.2          | 5.1      | 5.0      | 4.0      |
+|----------------------------------------------------------------------------------------------------------|:------------:|:------------:|:------------:|:------------:|:------------:|
+| [执行计划缓存](/sql-prepare-plan-cache.md)                                                   | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 |
```
Comment (TomShawn) [inline basic-features.md:71]:
```suggestion
| [执行计划缓存](/sql-prepare-plan-cache.md)                                                   | Y | 实验特性 | 实验特性 | 实验特性 | 实验特性 |
```

File: basic-features.md [hunk 2]
```diff
@@ -10,166 +10,166 @@ aliases: ['/docs-cn/dev/basic-features/']
...
+| [慢查询日志](/identify-slow-queries.md)                                                              | Y            | Y            | Y            | Y            | Y            |
+| [TiUP 部署](/tiup/tiup-overview.md)                                                                | Y            | Y            | Y            | Y            | Y            |
+| Ansible 部署                                                                                       | N            | N            | N            | N            | 已废弃   |
+| [Kubernetes operator](https://docs.pingcap.com/tidb-in-kubernetes/stable)                                      | Y            | Y            | Y            | Y            | Y            |
+| [内置物理备份](/br/backup-and-restore-use-cases.md)                                          | Y            | Y            | Y            | Y            | Y            |
+| Top SQL                                                                                                  | Y            | Y            | N            | N            | N            |
```
Comment (TomShawn) [inline basic-features.md:168]:
```suggestion
| Top SQL                                                                                                  | N            | N            | N            | N            | N            |
```

File: basic-features.md [hunk 3]
```diff
@@ -10,166 +10,169 @@ aliases: ['/docs-cn/dev/basic-features/']
...
-| [Change data capture (CDC)](/ticdc/ticdc-overview.md)                                                    | Y            | Y            | Y            | Y            |
+| 数据导入和导出                                                                               | 5.3          | 5.2          | 5.1      | 5.0      | 4.0      |
+|----------------------------------------------------------------------------------------------------------|:------------:|:------------:|:------------:|:------------:|:------------:|
+| [快速导入 (TiDB Lightning)](/tidb-lightning/tidb-lightning-overview.md)                             | Y            | Y            | Y            | Y            | Y            |
+| mydumper 逻辑导入                                                                                  | 已废弃   | 已废弃   | 已废弃   | 已废弃   | 已废弃   |
+| [Dumpling 逻辑导入](/dumpling-overview.md)                                                         | Y            | Y            | Y            | Y            | Y            |
+| [事务 `LOAD DATA`](/sql-statements/sql-statement-load-data.md)                                  | Y            | Y            | Y            | Y            | N            |
```
Comment (IANTHEREAL) [inline basic-features.md:152]: 为什么 4.0 不支持 load data？ 这个是按照什么定义的呀

**Review by lilyjazz — APPROVED** [review]

File: basic-features.md (hunk 3 above)
Comment (TomShawn) [reply basic-features.md:151]: @bb7133 PTAL

**Comment by ThomasYYYY** [comment]
LGTM

**Review by ThomasYYYY — APPROVED** [review]
LGTM

File: basic-features.md (hunk 3 above)
Comment (tangenta) [reply basic-features.md:151]: 这个是指 4.0 的 load data 是不保证原子性的，这个行为在 https://github.com/pingcap/tidb/pull/18807 中修复。

Comment (IANTHEREAL) [reply basic-features.md:151]: 可以标注一下，要不会有误解

Comment (IANTHEREAL) [reply basic-features.md:151]: Rest LGTM

Comment (qiancai) [reply basic-features.md]: @IANTHEREAL @tangenta 已添加注解，PTAL 谢谢

[^3]: 对于 TiDB v4.0，事务 `LOAD DATA` 不保证原子性。

**Comment by IANTHEREAL** [comment]
LGTM

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**